Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -173,14 +173,14 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
                 return;
             } else if (command.command === 'C_Cpp.CreateDeclarationOrDefinition' && (command.arguments ?? []).length === 0) {
                 command.arguments = ['codeAction']; // We report the sender of the command
-            } else if (command.command === "C_Cpp.SelectDefaultCompiler") {
+            } else if (command.command === "C_Cpp.SelectIntelliSenseConfiguration") {
```

Comment:
I think we still need to keep the old command around for this code action on the std headers.

Reply:
That code action now invokes the new quick pick (with config providers and compile commands).

Reply:
Yeah, we want that -- if the config provider or compile commands are set and system headers are not found, we want to have the config providers and compile commands in the quick pick if they're available and not just the compilers.

Reply:
We need to keep the "select a compiler" quick pick (for now) for the walkthrough experiment.